lua/starfall/libs_sv/ply.lua

```diff
@@ -0,0 +1,241 @@
+    
+    local ply = getPly( self )
+    ply:SetWalkSpeed( walkSpeed )
+end
+
+
+--- Sets the crouch speed multiplier of a player. This is a number from 0-1. Default is 0.3.
+-- @param number Multiplier
+function player_methods:setCrouchSpeedMultiplier( walkSpeed )
+    if not canUsePlyCore( instance.player ) then SF.Throw( "You must be an admin to use this function!", 2 ) end
+    
+    checkluatype( walkSpeed, TYPE_NUMBER )
+    checktype( self, ply_meta )
+    
+    local ply = getPly( self )
+    ply:SetCrouchedWalkSpeed( walkSpeed )
+end
+
+
+--- Sets the running speed of a player. This is when you're holding SHIFT. Default value is 600.
+-- @param number RunSpeed
+function player_methods:setRunSpeed( runSpeed )
+    if not canUsePlyCore( instance.player ) then SF.Throw( "You must be an admin to use this function!", 2 ) end
+    
+    checkluatype( runSpeed, TYPE_NUMBER )
+    checktype( self, ply_meta )
+    
+    local ply = getPly( self )
+    ply:SetRunSpeed( runSpeed )
+end
+
+
+--- Sets the gravity multiplier of a player. Passing 0 will reset it to its default value of 1.
+-- @param number Gravity
+function player_methods:setGravity( gravity )
+    if not canUsePlyCore( instance.player ) then SF.Throw( "You must be an admin to use this function!", 2 ) end
+    
+    checkluatype( gravity, TYPE_NUMBER )
+    checktype( self, ply_meta )
+    
+    local ply = getPly( self )
+    ply:SetGravity( gravity )
+end
+
+
+--- Sets the position of a player.
+-- @param vector Position
+function player_methods:setPos( pos )
+    if not canUsePlyCore( instance.player ) then SF.Throw( "You must be an admin to use this function!", 2 ) end
+    
+    checktype( self, ply_meta )
+    
+    local ply = getPly( self )
+    local position = vunwrap( pos )
+    ply:SetPos( position )
+end
+
+
+--- Sets the eye angles of a player.
+-- @param angle EyeAngles
+function player_methods:setEyeAngles( ang )
+    if not canUsePlyCore( instance.player ) then SF.Throw( "You must be an admin to use this function!", 2 ) end
+    
+    checktype( self, ply_meta )
+    
+    local ply = getPly( self )
+    local angle = aunwrap( ang )
+    ply:SetEyeAngles( angle )
+end
+
+
+--- Strips all ammo from a player.
+function player_methods:stripAmmo()
+    if not canUsePlyCore( instance.player ) then SF.Throw( "You must be an admin to use this function!", 2 ) end
+    
+    checktype( self, ply_meta )
+    
+    local ply = getPly( self )
+    ply:StripAmmo()
+end
+
+
+--- Sets the ammo of the player. Ammo ID can either be a number or a string. Refer to the keys given from the Player:getAmmo() table for numeric IDs.
+-- @param any AmmoID
+-- @param number Amount
+function player_methods:setAmmo( ammoId, amount )
+    if not canUsePlyCore( instance.player ) then SF.Throw( "You must be an admin to use this function!", 2 ) end
+    if type( ammoId ) ~= "string" and type( ammoId ) ~= "number" then SF.Throw( "Ammo ID must either be a string or a number!", 2 ) end
```

Comment:
```suggestion
    if type( ammoId ) ~= "string" and type( ammoId ) ~= "number" then 
        SF.Throw( "Ammo ID must either be a string or a number!", 2 ) 
    end
```
this line is starting to be on the long side, writing it as multiple lines like so would be preferable 
```